dhall-lsp-server/src/Dhall/LSP/Server.hs

```diff
@@ -78,7 +79,7 @@ run = withLogger $ \ioLogger -> do
               ]
         }
 
-  let staticHandlers =
+  let staticHandlers _clientCapabilities =
```

Comment:
Do we need to respect the client's capabilities indicated by this argument?

Reply:
The [documentation](https://hackage.haskell.org/package/lsp-2.3.0.0/docs/Language-LSP-Server.html#t:ServerDefinition) heavily implies it's optional and the handlers didn't previously use them, so I think we can ignore it.